graph/assemblygraph.cpp

```diff
@@ -51,6 +51,7 @@
 
 AssemblyGraph::AssemblyGraph()
         : m_sequencesLoadedFromFasta(NOT_READY)
+        , m_currentScope(graph::Scope::wholeGraph())
```

Comment:
I do not think this is a proper approach. AssemblyGraph is a data structure and it should not "know" about the drawing scope, so this way you're leaking the abstraction through unrelated data structure.

Certainly, we do have some legacy issues (e.g. global settings state that is changed in different places, but we should improve the situation, and not make it worse).